include/matx/generators/generator1d.h

```diff
@@ -54,6 +54,9 @@ namespace matx
             return f_(pp_get<Dim>(indices...));
           }
 
+        template <typename Task>
+        __MATX_INLINE__ void apply_dep_to_task([[maybe_unused]] Task &&task, [[maybe_unused]] int perm=1) const noexcept { }
```

Comment:
So this operator is defined per operator, and is STF specific ? it's not part of the executor nor relying on overloads / traits ?